.travis.yml

```diff
@@ -11,8 +11,8 @@ env:
     - SCVERSION="stable"
 
 before_install:
-    - docker pull quay.io/coreos-assembler/coreos-assembler:latest
-    - docker build -t coreos-assembler-test -f Dockerfile.dev .
+    - docker pull registry.fedoraproject.org/fedora:31
```

Comment:
Do we even need this? Won't `docker build` already automatically pull the right image based on `FROM`?

(The reason I ask is that if this works, then we can just copy/paste this across all the branches and especially for the main branch it's one less thing to tweak every time we change `$releasever`).

Reply:
I don't think we need it as it will automatically pull based on FROM in the Dockerfile.